auto_submit/lib/service/approver_service.dart

```diff
@@ -32,36 +32,43 @@ class ApproverService {
   }
 
   /// Auto approves a pull request when the revert label is present.
-  Future<void> revertApproval(PullRequest pullRequest) async {
+  Future<void> revertApproval(gh.PullRequest pullRequest) async {
     final Set<String> approvedAuthorAssociations = <String>{'MEMBER', 'OWNER'};
 
     final String? author = pullRequest.user!.login;
     final String? authorAssociation = pullRequest.authorAssociation;
+
+    log.info('Attempting to approve revert request by author $author, authorAssociation $authorAssociation.');
+
     final List<String> labelNames =
-        (pullRequest.labels as List<IssueLabel>).map<String>((IssueLabel labelMap) => labelMap.name).toList();
+        (pullRequest.labels as List<gh.IssueLabel>).map<String>((gh.IssueLabel labelMap) => labelMap.name).toList();
+
+    log.info('Found labels $labelNames on this pullRequest.');
 
     if (labelNames.contains(Config.kRevertLabel) &&
         (config.rollerAccounts.contains(author) || approvedAuthorAssociations.contains(authorAssociation))) {
+      log.info(
+          'Revert label and author has been validated. Attempting to approve the pull request. $pullRequest by $author');
```

Comment:
`$pullRequest` will not give you the pull request info. Instead, try adding the pull request repo and number.

Reply:
But I want to see what the author association is. Won't we get that repo info from this as well since it is a member of the class?

Reply:
Line 41 already returns the association info?
What I meant here is `pullRequest` doesn't have a default `toString()` method override, it will just print message `an instance of Pull Request` in the log without giving you any extra useful info.

Reply:
Changed the log message.